src/helpers/database.py

```diff
@@ -0,0 +1,94 @@
+from sqlalchemy import text
+from sqlalchemy.engine import Engine
+from src.helpers.config import check_db_connection, logger
+from src.helpers.helper_functions import read_sql_file
+
+
+class Database:
```

Comment:
Same comment as for the BlockchainData class.

Also, is there a reason one class is called `BlockchainData` while the other is just `Database`? Would it make sense to make the names uniform or is the choice of names deliberate?

Reply:
It was deliberate. Should I just rename BlockchainData to Blockchain then?